johannes_mueller/entry_sphinx/Readme.md

```diff
@@ -0,0 +1,307 @@
+# Automated package documentation with Sphinx
+
+## Introduction
+We've all been there. We have written this wonderful piece of software that will essentially solve science's most pressing problems, the only *minor* thing left to do is propper documentation. And this, although it's only the lat few meters until the finish lines, is where good projects often turn into bad projects.
+
+<img src="https://user-images.githubusercontent.com/38459088/143226848-c0765ba7-43bc-4b9d-99ce-8426f2177cfa.jpg" width="400" height="400">
+
+Enter Sphinx: [Sphinx](https://www.sphinx-doc.org/en/master/) is a tool that can *automatically* generate documentation in various formats (html, pdf, etc) based on the docstrings in your code. Popular examples for documentation pages that have been built with Sphinx, are [scitkit-learn](https://scikit-learn.org/stable/index.html) or [scikit-image](https://scikit-image.org/).
+
+This blog entry will walk you through the generation of documentation html pages for an exemplary github library using Sphinx and a few extensions.
+
+Blogs on this topic will cover:
+* [Getting started](https://github.com/BiAPoL/blog/blob/sphinx-entry/johannes_mueller/entry_sphinx/Readme.md#getting-started)
+* [Running Sphinx](https://github.com/BiAPoL/blog/blob/sphinx-entry/johannes_mueller/entry_sphinx/Readme.md#running-sphinx)
+* [Hosting pages on Github](https://github.com/BiAPoL/blog/blob/sphinx-entry/johannes_mueller/entry_sphinx/Readme.md#host-your-documentation-on-github)
+
+
+## Getting started
+
+First off, clone the example repository from github into an empty directory with [Github Desktop](https://desktop.github.com/) or using
+
+```
+git clone https://github.com/BiAPoL/example-repo-sphinx.git
+cd example-repo-sphinx/
+```
+
+*Optional, but recommended*: Create a new environment for this tutorial using 
+
+```
+conda create -n sphinx-tutorial
+conda activate sphinx-tutorial
+```
+
+The repository has a typical structure: It consists of a top-level module (`verygoodpackage`) and a submodule (`subpackage_a`) that consists function definitions in `_operations.py`. There are two functions (`add` and `subtract`) which we would like to document:
```

Comment:
missing of after the 2nd consists: (`subpackage_a`) that consists function definitions